src/cpp/libfastdds.version

```diff
@@ -0,0 +1,4 @@
+CODEABI_1.0 {
+	global: *eprosima*;*MD5*;
```

Comment:
May I ask why is *dds* not longer included ?

Reply:
The dds namespace is no longer exported as the API PSM has been removed. @MiguelCompany, do you agree?